### PR TITLE
Maintain map position when loading a new layer

### PR DESCRIPTION
**Pull request by reed1**
For #345

### CODE REVIEW

**Comment by ericboucher** [comment]
@reed1 I just merged a big change to use maplibre instead of mapbox and it seems to have broken your change. Sorry!

**Comment by reed1** [comment]
@ericboucher @wadhwamatic It's updated now, still works with maplibre, it's just removing the center and zoom from map initial props. A better solution might be patching the library or adding position in state variable and watch the changes? but I think this will help for the time being

**Review by wadhwamatic — COMMENTED** [review]
Tested and works as expected. Thanks!
Let's wait for review from @Ry-DS or @ericboucher before merging

**Review by ericboucher — APPROVED** [review]

**Pull request merged**
